packages/docs/docs/sidekicks-quickstart.md

```diff
@@ -27,212 +34,193 @@ At the end of this quickstart you will:
+hitting Ctrl-C), the Sidekick reverts back to the most recently
+deployed version (from `fixie deploy`). Note that you need to `fixie
+deploy` your Sidekick in order for it to run in the cloud.
 
-```terminal
-sarah@computer % FIXIE_API_URL='https://console.fixie.ai' npx @fixieai/fixie deploy
-🦊 Deploying agent sarah/sidekick-acme...
-👽 Updating agent sarah/sidekick-acme...
-✔ Revision a9b5e04c was deployed to https://console.fixie.ai/agents/sarah/sidekick-acme
-```
-
-#### Troubleshooting: Verify the Sidekick Builds
-
-If the above deploy step failed for some reason, it is a good idea to build our sidekick locally to see if there are any errors raised. Building is done from the terminal:
-
-```terminal
-npm build
-```
-
-_TODO Matt: Is this still needed?_
+As we make changes to our sidekick, we can simply stop serving our sidekick with `Control+C` and then serve up our new changes with the `serve` command as we did above.
 
-## Step 3: Converse with Sidekick
 
-<img src={Step3Profit} alt="Fixie profile page where you can get your API key." width="300"/>
+## Sending messages to your Sidekick
 
-We've got our Sidekick deployed. Let's test it out to see what it can do!
+The Fixie Console page gives you a simple web interface to interact with your
+Sidekick, but you're not limited to this interface. You can chat with your
+Sidekick directly via a REST API, or embed the Sidekick chat UI in your own
+web app.
 
-We are going to test the sidekick using four different methods.
+### Method 1: Via the REST API
 
-### Method 1: Via curl
+> TODO(matt): Figure out the right invocation for this.
 
-First up, let's ask our Sidekick a question using curl. From your terminal:
+First up, let's ask our Sidekick a question through the Fixie REST API, using
+`curl`. From your terminal:
 
 ```bash
 curl 'https://console.fixie.ai/api/agents/<your user name>/<your sidekick name>' \
   -d '{ "message": {"text": "What can you do?" }}' \
   -H 'Authorization: Bearer <your Fixie API key>' \
   -H 'Content-Type: application/json'
+```
 
--- For example:
+For example:
 
-curl 'https://console.fixie.ai/api/agents/sarah/sidekick-acme' \
+```bash
+curl 'https://console.fixie.ai/api/agents/sarah/fixie-sidekick-template' \
   -d '{ "message": {"text": "What can you do?" }}' \
   -H 'Authorization: Bearer FmEEMtjcHLfNGPrLhRQwQfwG9Li...' \
   -H 'Content-Type: application/json'
 ```
 
-_TODO Matt: this is currently broken. not sure what the plan is to fix._
-
-This should display something like this:
-
-TODO
-
-### Method 2: Via the Fixie Dashboard
+### Method 2: Via the Fixie Console
 
 - In your browser navigate to the [Fixie dashboard](https://console.fixie.ai/).
-- Click on your agent.
-- Enter a question for the sidekick. e.g. "What can you do?"
+- Click on your Sidekick.
+- Enter a question for the Sidekick. e.g. "What can you do?"
 
-Note: you can also access your agent directly at:
+You can also access your agent directly at:
```

Comment:
This section feels redundant. I don't think we need to both provide a direct link to the dev console, and also tell people how to go to the front door of the dev console and then click through it.